src/ex3/CLI.js

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
+
+import { ItemManager } from './ItemManager.js';
+import { FailedFetchPokemonError, PokemonAlreadyInError} from './Errors.js';
+import { Command } from "commander";
+import inquirer from 'inquirer';
+import chalk from 'chalk';
+import figlet from 'figlet';
+import gradient from 'gradient-string';
+import chalkAnimation from 'chalk-animation';
+import asciify from 'asciify-image';
+
+
+const itemManager = new ItemManager();
+const program = new Command();
+const sleep = (ms = 1200) => new Promise((r) => setTimeout(r, ms));
+const options = {
+    fit: 'none',
+    width: 30,
+    height: 20
+}
+
+
+async function addTask(task) {
+    if (task === '') {
+        console.log("you forgot to type the task!");
+        return;
+    }
+    const taskSplit = task.split(',');
+    if (isNaN(taskSplit[0])) {
+        itemManager.addRegTask(task);
+        console.log(chalk.bgGreenBright("new task added successfully!"));
+        return;
+    }
+    try {
+        const pokemons = await itemManager.getPokemon(taskSplit);
+        if (pokemons.length === 1) {
+            const pokemon = await itemManager.addPokemon(pokemons[0]);
```

Comment:
again regarding separations you have few logic pieces here that are regardless to the client you'll use, you'll need to fetch pokemons and check response, or loop on a list of pokemons no matter if your client will be web app or cli